editor/src/clj/editor/pipeline/texture_set_gen.clj

```diff
@@ -96,8 +96,12 @@
                                        hull-vertex-count (sprite-trim-mode->hull-vertex-count sprite-trim-mode)]
                                    (TextureSetGenerator/buildConvexHull buffered-image hull-vertex-count)))
                                images)
+        hull-sizes (map (fn [{:keys [path sprite-trim-mode] :as _image}]
+                          (sprite-trim-mode->hull-vertex-count sprite-trim-mode))
+                         images)
+        use-geometries (if (nil? (some (complement zero?) hull-sizes)) 0 1)
```

Comment:
I think this would be equivalent?
```clj
use-geometries (if (some #(not= :sprite-trim-mode-off (:sprite-trim-mode %)) images) 1 0)
```

Reply:
Thanks! So `#` creates something like a lambda from another function `%` is the argument of this new function, and `nil == false` and `not nil == true`, right? I've done this refactoring here and also for tile maps. And I also tested this now for tile maps as well as for atlas. And I moved hull-sizes calculation into a `let`. Check it out :)

Reply:
Oh, I see now, I don't even need hull-sizes... :)

Reply:
I removed hull-sizes calculations. And it seems I need to re-understand the `%` part :D

Reply:
You've got it! 😄